electron/src/lib/openGraph.ts

```diff
@@ -150,7 +149,14 @@ export const axiosWithContentLimit = (config: AxiosRequestConfig, contentLimit:
         response.data.on('error', reject);
         response.data.on('end', () => resolve(partialBody));
       })
-      .catch(error => (axios.isCancel(error) ? Promise.resolve('') : Promise.reject(error)));
```

Comment:
This was not using the outer `Promise` and thus created an uncaught promise rejection.